packages/components/src/popover/style.scss

```diff
@@ -149,9 +149,13 @@ $arrow-size: 8px;
 
 	.components-popover:not(.is-mobile) & {
 		position: absolute;
-		min-width: 260px;
 		height: auto;
 		overflow-y: auto;
+
+		// Let the menu scale to fit items.
+		width: auto;
```

Comment:
These are literally out of alphabetical order and I find that makes them so tough to read. Not part of the coding standards though, so I gotta just live with it I guess 🤷‍♂️